src/si/volume_rate.rs

```diff
@@ -77,6 +77,7 @@ quantity! {
         @decaliter_per_second: prefix!(milli) * prefix!(deca); "daL/s", "decaliter per second",
             "decaliters per second";
         @liter_per_second: prefix!(milli); "L/s", "liter per second", "liters per second";
+        @liter_per_minute: prefix!(milli) / 60.; "L/min", "liter per minute", "liters per minute";
```

Comment:
Move this line down to a new group underneath `@yoctoliter_per_second`. Add a test case as well.

Reply:
EDIT: change the conversion factor to `prefix!(milli) / 6.0_E1;`.

Reply:
Done!